### PR TITLE
fix: increase log level for reindexing problems

### PR DESCRIPTION
**Pull request by hsiliev**
Use WARNING for stopped reindexing process

Solves: PZ-3493

### CODE REVIEW

**Review by infonl-marcel — APPROVED** [review]

**Review by bas-info-nl — APPROVED** [review]

File: src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt [hunk 1]
```diff
@@ -327,9 +327,12 @@ class IndexingService @Inject constructor(
     private fun logTypeMessage(objectType: ZoekObjectType, message: String) =
         LOG.info("[$objectType] $message")
 
-    private fun logTypeError(objectType: ZoekObjectType, error: Throwable) =
+    private fun logTypeWarning(objectType: ZoekObjectType, error: Throwable) =
```
Comment (edgarvonk) [inline src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt:330]: Which `type` are you referring to? I don't understand the word `type` here. Why not just `logIndexingError` ? The caller of this function is not interested whether it is logged as at the 'WARNING' log level. The function is about logging an error during indexing right?

Comment (edgarvonk) [reply src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt:330]: Not sure we even need this function at all to be honest. It only confuses me. I would just log right there where the issue was.

File: src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt [hunk 2]
```diff
@@ -327,9 +327,12 @@ class IndexingService @Inject constructor(
...
         LOG.log(Level.WARNING, "[$objectType] Error during indexing", error)
 
+    private fun logTypeWarning(objectType: ZoekObjectType, message: String) =
```
Comment (edgarvonk) [inline src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt:333]: Same here. I would remove this entire function. I don't see the added value at all.

**Review by edgarvonk — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt (hunk 1 above)
Comment (hsiliev) [reply src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt]: Removed log functions

**Review by edgarvonk — APPROVED** [review]

**Pull request merged**